janus-gateway/janus.conf

```diff
@@ -1 +1 @@
-u janus 531 "Janus WebRTC gateway user" /var/lib/janus /bin/bash
+u janus 531 "Janus WebRTC gateway user"
```

Comment:
Making the home directory go away if it is actually unused is fine, but should we also assign a dummy shell like `/bin/true`? Or in the brave new world of systemd managed user accounts is that not helful?

Reply:
Turns out, you end up with /usr/bin/nologin if you leave it empty here. I guess you can trust the defaults in this brave new world 😄